docs/conf/prague/2020/news/prg-covid-19-update-02.rst

```diff
@@ -16,7 +16,7 @@ The most recent COVID-19 update is:
 
 Hello everyone,
 
-Since our :doc:`first announcement </conf/prague/2020/news/covid-19-update-01>` two weeks ago, we've been following the COVID-19 updates from the relevant authorities. At the time of writing this update, Czech Republic is still under lockdown and the government predicts that this situation will continue for at least a month.
+Since our :doc:`first announcement </conf/prague/2020/news/prg-covid-19-update-01>` two weeks ago, we've been following the COVID-19 updates from the relevant authorities. At the time of writing this update, Czech Republic is still under lockdown and the government predicts that this situation will continue for at least a month.
```

Comment:
Not precisely true, https://www.unian.info/world/covid-19-worldwide-czech-republic-opens-borders-10971311.html
https://www.reuters.com/article/us-health-coronavirus-czech/czechs-lift-ban-on-movement-and-travel-speed-up-reopening-shops-as-new-covid-19-cases-slow-idUSKCN22531H

```suggestion
Since our :doc:`first announcement </conf/prague/2020/news/prg-covid-19-update-01>` two weeks ago, we've been following the COVID-19 updates from the relevant authorities. At the time of writing this update, Czech Republic is still under quarrantine.
```

Reply:
I believe this is just updating the URL in the old post. We don't want ot change the actual context.